107/src/test/java/org/ehcache/jsr107/Eh107CacheTypeTest.java

```diff
@@ -79,9 +79,6 @@ public void testRunTimeTypeSafety() throws Exception {
 
     try {
       cacheManager.getCache("cache1");
-      fail("Caches with runtime types should throw illegal argument exception when different types are used in getcache");
```

Comment:
This seems really broken... am I missing something here?

Reply:
See jsr107/jsr107spec#340 for the details behind this change. It effectively covers up a hole in the API.